youtube_dl/extractor/orf.py

```diff
@@ -365,7 +379,8 @@ def _real_extract(self, url):
 
             duration = float_or_none(data['duration'], 1000)
 
-            video = data['sources']['q8c']
+            sources = data['sources']
+            video = sources.get('default') or data['sources']['q8c']
```

Comment:
There are more formats (https://fm4.orf.at/stories/2956045/), all should be extracted.